pkg/kubelet/dockershim/docker_sandbox_test.go

```diff
@@ -156,17 +156,14 @@ func TestSandboxStatus(t *testing.T) {
 	assert.Error(t, err, fmt.Sprintf("status of sandbox: %+v", statusResp))
 }
 
-// TestSandboxHasLeastPrivilegesConfig tests that the sandbox is set with no-new-privileges
-// and it uses runtime/default seccomp profile.
-func TestSandboxHasLeastPrivilegesConfig(t *testing.T) {
+// TestSandboxDockerConfig tests generating the sandbox docker config
+func TestSandboxDockerConfig(t *testing.T) {
```

Comment:
As we are moving the test `TestSandboxHasLeastPrivilegesConfig` to a separate file: docker_sandbox_linux_test.go,  do we still need `TestSandboxDockerConfig`, since the logic in the test is simple and was covered by `TestSandboxStatusAfterRestart` ?

Reply:
Removing it may be a better choice.